app/src/main/java/com/mapbox/services/android/navigation/testapp/activity/OfflineRegionDownloadActivity.kt

```diff
@@ -208,8 +211,8 @@ class OfflineRegionDownloadActivity : AppCompatActivity(), RouteTileDownloadList
         }
 
         if (ContextCompat.checkSelfPermission(
-                this, WRITE_EXTERNAL_STORAGE
-            ) != PERMISSION_GRANTED
+            this, WRITE_EXTERNAL_STORAGE
+        ) != PERMISSION_GRANTED
```

Comment:
As discussed internally noticed a formatting mismatch between `ktlintFormat` and `Code | Reformat` `⌥⌘L` from the IDE that we should fix (I personally rely on `⌥⌘L` heavily 😅) 👀 

```kotlin
private fun onDownloadClick() {
        // todo check that download is less than 1.5 million square kilometers
        if (!downloadButtonEnabled) {
            return
        }
        if (ContextCompat.checkSelfPermission(
            this, WRITE_EXTERNAL_STORAGE
        ) != PERMISSION_GRANTED
        ) {
            ActivityCompat.requestPermissions(this, arrayOf(WRITE_EXTERNAL_STORAGE), 1)
        } else {
            downloadMapsRegion()
        }
    }
```

vs

```kotlin
private fun onDownloadClick() {
        // todo check that download is less than 1.5 million square kilometers
        if (!downloadButtonEnabled) {
            return
        }
        if (ContextCompat.checkSelfPermission(
                this, WRITE_EXTERNAL_STORAGE
            ) != PERMISSION_GRANTED
        ) {
            ActivityCompat.requestPermissions(this, arrayOf(WRITE_EXTERNAL_STORAGE), 1)
        } else {
            downloadMapsRegion()
        }
    }
```

Which BTW I personally prefer the latter as 

```kotlin
if (ContextCompat.checkSelfPermission(
        this, WRITE_EXTERNAL_STORAGE
    ) != PERMISSION_GRANTED
) {
```

seems more natural.

```kotlin
if (ContextCompat.checkSelfPermission(
    this, WRITE_EXTERNAL_STORAGE
) != PERMISSION_GRANTED
) {
```

is confusing IMO.

The same thing happens to a bunch of other files 👇